apps/web/src/content/blog/en/transform-lovable-dev-app-to-mobile-with-capacitor.md

```
@@ -86,37 +112,30 @@ Before we can work with your code, you'll need a code editor. We recommend [Curs
-   - Access to GPT-4 and Claude models
-   - Faster response times
-   - Priority support
+For the best experience, configure Cursor before you start:
 
+1. **Buy a Cursor Plan** — While Cursor offers a free tier, a Pro plan ($20/month) gives you unlimited AI completions, access to Claude and GPT-4, and command execution
 2. **Open Cursor Settings** by pressing `Command+,` (Mac) or `Ctrl+,` (Windows)
 
 ![Cursor Settings](/cursor_settings.webp)
 
-3. **Enable AI Models** - Make sure AI features are enabled:
+3. **Enable AI Models** — Make sure AI features are enabled:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)